ray-operator/controllers/ray/common/association.go

```diff
@@ -74,6 +74,16 @@ func RayClusterWorkerPodsAssociationOptions(instance *rayv1.RayCluster) Associat
 	}
 }
 
+func RayClusterRedisPodAssociationOptions(instance *rayv1.RayCluster) AssociationOptions {
```

Comment:
```suggestion
func RayClusterRedisCleanupPodAssociationOptions(instance *rayv1.RayCluster) AssociationOptions {
```
Just to clarify that the labels are only applied to the cleanup jobs and pods, not Redis instances.

Reply:
Thank you for pointing it out!